Tools/scripts/build_options.py

```diff
@@ -54,6 +54,10 @@ def __init__(self,
     Feature('Telemetry', 'HIGHLAT2', 'HAL_HIGH_LATENCY2_ENABLED', 'Enable HighLatency2 Support', 0, None),
     Feature('Telemetry', 'HOTT', 'HAL_HOTT_TELEM_ENABLED', 'Enable HOTT Telemetry', 0, None),
     Feature('Telemetry', 'SPEKTRUM', 'HAL_SPEKTRUM_TELEM_ENABLED', 'Enable Spektrum Telemetry', 0, None),
+    Feature('Telemetry', 'FrSky', 'AP_FRSKY_ENABLED', 'Enable FrSky Telemetry', 0, None),
+    Feature('Telemetry', 'FrSky D', 'AP_FRSKY_D_ENABLED', 'Enable FrSkyD Telemetry', 0, 'AP_FRSKY_ENABLED'),
+    Feature('Telemetry', 'FrSky SPort', 'AP_FRSKY_SPORT_ENABLED', 'Enable FrSkySPort Telemetry', 0, 'AP_FRSKY_ENABLED'),
+    Feature('Telemetry', 'FrSky SPort PassThrough', 'AP_FRSKY_SPORT_PASSTHROUGH_ENABLED', 'Enable FrSkySPort PassThrough Telemetry', 0, 'AP_FRSKY_SPORT_ENABLED'),  # noqa
```

Comment:
I think CRSF passthrough requires SPort passthrough so you would have to add a dependency here

Reply:
Added the dep.